media/css/m24/flag.scss

```diff
@@ -63,6 +63,21 @@
     &:active {
         background-color: $m24-color-dark-mid-gray;
     }
+
+    // High Contrast Mode Support: https://bugzilla.mozilla.org/show_bug.cgi?id=1936865
+    @media (forced-colors) {
+        background-color: ButtonFace;
+        color: ButtonText;
+
+        &:focus {
+            outline-color: CanvasText;
+        }
+    }
+
+    // Focused control is not hidden under sticky nav: https://bugzilla.mozilla.org/show_bug.cgi?id=1936862
+    &:focus {
+        scroll-margin-top: var(--sticky-nav-height);
```

Comment:
scroll margin experiment, don't think I'm going to use it, only seems to work with no banner and it doesn't seem worth adding up the nav & occasional banner heights

I think that leaves me with a z-index override option? open to other ideas